yarn.lock

```diff
@@ -4507,6 +4507,11 @@ clsx@^1.2.1:
   resolved "https://registry.yarnpkg.com/clsx/-/clsx-1.2.1.tgz#0ddc4a20a549b59c93a4116bb26f5294ca17dc12"
   integrity sha512-EcR6r5a8bj6pu3ycsa/E/cKVGuTgZJZdsyUYHOksG/UHIiKfjxzRxYJpyVBwYaQeOvghal9fcc4PidlgzugAQg==
 
+clsx@^2.0.0:
```

Comment:
Would make sense to first allow the other MUI packages to update it on their end, before doing the same on our end to avoid two different majors being resolved/used.

Reply:
https://github.com/mui/material-ui/pull/38121

Reply:
Thank you for clarifying that it has already been merged.
We'll just wait for the `MUI Core` sync PR and this should be ready to merge after that. 👌